### DISCUSSION TITLE
Great initiative! - please keep it up to date

### DISCUSSION THREAD

**Discussion by Nir-Az** [libusb/libusb-cmake#28] [General]
We used to maintain a custom CMake project for libusb and it was hard keeping it up to date as the files kept changing and we were missing the configure file.
Thanks for opening this repo, this helps a lot.
Managed to update to latest version in 1 hour of work! :)

Please keep it maintained, it is very useful
😎

**Comment by Youw** [comment]
> please keep it up to date

We are.

If I missed anything obvious - let me know.
